packages/testing/ts/__tests__/e2e.nonQv.test.ts

```diff
@@ -701,12 +697,14 @@ describe("e2e tests with non quadratic voting", function test() {
     });
 
     it("should join two users", async () => {
-      for (let index = 0; index < users.length; index += 1) {
+      // we cannot use Promise.all( users.map( ... ) ) because they will run concurrently
```

Comment:
es-lint was giving an error asking me to change from
```javascript
for (let index = 0; index < users.length; index += 1) {
``` 

to

```javascript
for (const user of users) {
```

and later on even to
```javascript
Promise.all( users.map( ... ) )
```


The problem with `users.map( ... )` is that it runs concurrently and not sequentially. This could cause non-deterministic errors between function executions. Remember that each user can join the poll but they need to do it sequentially.


I decided that the best way to solve this was to disable es-lint for this line and add a comment explaining why

Reply:
can't we keep it as  for (let index = 0; index < users.length; index += 1) as that's what we usually use?

Reply:
I really don't mind but the es-lint throws an error, to silence it we have 2 options:

```javascript
// eslint-disable-next-line @typescript-eslint/prefer-for-of
for (let i= 0; i< users.length; i+= 1) {
```

```javascript
// eslint-disable-next-line no-restricted-syntax
for (const user of users) {
```

Let me know which one you prefer 😊 @ctrlc03 @0xmad